config/default.yml

```
@@ -570,6 +570,12 @@ Rails/Validation:
   Include:
     - app/models/**/*.rb
 
+Rails/WhereNot:
+  Description: 'Use `where.not(...)` instead of manually constructing negated SQL in `where`.'
+  StyleGuide: 'https://rails.rubystyle.guide/#where-not'
+  Enabled: true
+  VersionAdded: '2.7'
```

This looks good. Can you update this one and rebase with the latest master branch?

```suggestion
  VersionAdded: '2.8'
```